__tests__/app.test.js

```diff
@@ -79,6 +92,57 @@ describe('/api',()=>{
                 })
             })
         })
+        test('GET:200 Should respond with array of article objects sort by created_at and default descending order',()=>{
+            return request(app).get('/api/articles').expect(200).then(({body})=>{
+                expect(body.articles).toBeSortedBy('created_at',{descending:true})
+            })
+        })
+        test('GET:200 Should respond with array of article objects with correct datatypes',()=>{
+            return request(app).get('/api/articles').expect(200).then(({body})=>{
+                body.articles.forEach((article)=>{
+                    expect(typeof article.author).toBe('string')
+                    expect(typeof article.title).toBe('string')
+                    expect(typeof article.article_id).toBe('number')
+                    expect(typeof article.topic).toBe('string')
+                    expect(typeof article.created_at).toBe('string')
+                    expect(typeof article.votes).toBe('number')
+                    expect(typeof article.article_img_url).toBe('string')
+                    expect(typeof article.comment_count).toBe('string')
+                    
+                })
+            })
+        })
+    })
+    describe('/articles/:article_id/comments endpoints',()=>{
+        test('GET:200 Should return array of comments for the given article_id',()=>{
+            return request(app).get('/api/articles/3/comments').expect(200)
+            .then(({body})=>{
+                body.comments.forEach((comment)=>{
+                    expect(comment).toHaveProperty('comment_id');
+                    expect(comment).toHaveProperty('votes');
+                    expect(comment).toHaveProperty('created_at');
+                    expect(comment).toHaveProperty('author');
+                    expect(comment).toHaveProperty('body');
+                    expect(comment).toHaveProperty('article_id');
+                })
```

Comment:
once again you could be more specific here and also assert that the properties hold the right data type. I would highly recommend you look into another assertion to make the syntax of all tests that look like this nicer. Have a look at toMatchObject or objectContaining and see how they are used to check the structure of the returned data